ext/standard/tests/mail/mailhog_utils.inc

```diff
@@ -0,0 +1,66 @@
+<?php
+
+function searchEmailByToAddress (string $to)
+{
+    // Search the mail by $to
+    $c = curl_init();
+    curl_setopt($c, CURLOPT_URL, 'http://localhost:8025/api/v2/search?kind=to&query='.$to);
+    curl_setopt($c, CURLOPT_RETURNTRANSFER, true);
+    $res = curl_exec($c);
+    curl_close($c);
+
+    return json_decode($res, true);
+}
+
+function getHeaders ($response)
```

Comment:
we mostly try to follow PSR-12 in tests (not strictly but better not to do space before `(` in this case)